dapps/marketplace/src/pages/listings/_Search.js

```diff
@@ -260,16 +262,16 @@ require('react-styl')(`
           margin-bottom: 1rem
 
         .featured-categories-wrapper
-          margin: 0 -2rem
-          overflow-x: scroll
+          margin: 0 0rem
 
         .featured-categories
           display: inline-flex
-          padding: 0 2rem
+          margin: 0 -10px
+          flex-wrap: wrap;
```

Comment:
nit: the convention in our codebase is to leave the semicolons (e.g. character ";") out. please remove.